### ISSUE TITLE
GLBindings.Import problems

### ISSUE THREAD

**Issue by biffbaff64** [biffbaff64/LughSharp#2]
There is a probl;em in LibCore.Corelib.Graphics.OpenGL.GLBindings.

The method `void Import( GetProcAddressHandler loader )' fails to initialise the following GL functions:-

1. glGetnTexImage
2. glGetnUniformdv
3. glMultiDrawArraysIndirectCount
4. glMultiDrawElementsIndirectCount

Attempts to initialise these functions results in a crash.`

**Comment by damianday** [comment]
I suspect the API is wrong for them. 

At least for glMultiDrawArraysIndirectCount it is wrong according to khronos it should be this: 
https://registry.khronos.org/OpenGL-Refpages/gl4/html/glMultiDrawArraysIndirect.xhtml

**Comment by damianday** [comment]
Might be worth having a look at Silk.Net
They auto generate their GL bindings, maybe they have the same issue.

**Comment by biffbaff64** [comment]
Yeah, I did look at Silk, and OpenTK. I've gotten my stuff pretty much working now though. I'm working through a list of issues but it's improving rapidly.
